greenswitch/esl.py

```diff
@@ -458,6 +460,13 @@ def uuid_break(self):
             raise OutboundSessionHasGoneAway
         self.send('api uuid_break %s' % self.uuid)
 
+    def raise_if_disconnected(self):
+        """This function will raise the exception
+        esl.OutboundSessionHasGoneAway if the caller hung up the call
+        """
+        if self._lingering:
```

Comment:
I don't think it's correct to rely on lingering here since the user may not send linger command, we really need a flag to detect if the outbound session is still alive no matter the socket state is.

Reply:
I've updated to use the self._outbound_connected instead of self._lingering.